storage.rb

```diff
@@ -0,0 +1,70 @@
+require 'json'
+
+class Storage
+  BOOKS_FILE = './data/books.json'.freeze
+  PEOPLE_FILE = './data/people.json'.freeze
+  RENTALS_FILE = './data/rentals.json'.freeze
+
+  def initialize(app)
+    @app = app
+  end
+
+  def load_data
+    load_books(BOOKS_FILE, @app.books)
+    load_people(PEOPLE_FILE, @app.people, @app.classrooms)
+    load_rentals(RENTALS_FILE, @app.rentals, @app.books, @app.people)
+  end
+
+  def save_data
+    File.write(BOOKS_FILE, JSON.generate(@app.books.map(&:to_h)))
+    File.write(PEOPLE_FILE, JSON.generate(@app.people.map(&:to_h)))
+    File.write(RENTALS_FILE, JSON.generate(@app.rentals.map(&:to_h)))
+  end
+
+  private
+
+  def read_data(filename)
+    file = File.open(filename)
+    file_data = file.read
+    file.close
+    return JSON.parse(file_data) unless file_data.empty?
+
+    []
```

Comment:
- [ ] Please kindly adjust this logic, according to the requirement when one or more of the files are missing the app should not throw any error but perhaps return an empty array. I got this error when I deleted rentals.json file. Perhaps, this `return [] unless File.exists?("filename.json")` will resolve the issue. 


<img width="724" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/226581462-29376bb4-3190-466a-9479-d6b1028f674f.png">

Reply:
Thanks for your suggestion. I'll implement it right away.